azext_iot/_params.py

```diff
@@ -263,6 +263,18 @@ def load_arguments(self, _):
             help="Twin desired properties.",
         )
 
+    with self.argument_context("iot hub connection-string") as context:
+        context.argument(
+            "show_all",
+            options_list=["--show-all", "--all"],
+            help="Allow to show all shared access policies."
+        )
+        context.argument(
+            "include_eventhub",
+            options_list=["--include-eventhub", "--eh"],
```

Comment:
Instead of `--include-eventhub` let's do `--default-eventhub`

Reply:
updated